### PR TITLE
clean-up namedtuple (#64)

### LINKED ISSUE #64: native client: how to return multiple values?
## Description

In the current *native* client implementation we either return a single value or a list of values. I would find it more descriptive to return a dictionary with: 
```
key=parameter identifier
value=parameter value
```

@huard @davidcaron 
What do you think?

Here a link from a quick google search:
https://stackoverflow.com/questions/354883/how-do-you-return-multiple-values-in-python

### PR DESCRIPTION
**Pull request by cehbrecht**
## Overview

This PR is a follow-up of PR #74 to fix #64.

Changes:

* Using `birdy.utils.make_identifier` to make a valid Python method name.
* Moved `delist` to `birdy_utils` ... added test.
* Using `namedtuple._make`

## Related Issue / Discussion

#64 and PR 74.

## Additional Information

See also ticket #69 to handle invalid Python variables, functions.

`utils.make_identifier` is just a first attempt.

### CODE REVIEW

File: birdy/client/base.py [hunk 1]
```diff
@@ -214,8 +216,8 @@ def _execute(self, pid, **kwargs):
             raise
 
         # Output type conversion
-        output = namedtuple(pid, [o.identifier for o in resp.processOutputs])
-        return output(**{o.identifier: self._process_output(o, pid) for o in resp.processOutputs})
+        Output = namedtuple('Output', [make_identifier(o.identifier) for o in resp.processOutputs])
+        return Output._make([self._process_output(o, pid) for o in resp.processOutputs])
```
Comment (huard) [inline birdy/client/base.py]: I think you could just use `Output(*[...])`, no ?

File: birdy/utils.py [hunk 2]
```diff
@@ -9,3 +13,17 @@ def is_url(url):
         return False
     else:
         return True
+
+
+def make_identifier(name):
```
Comment (huard) [inline birdy/utils.py]: rename to `sanitize` ?

File: birdy/client/base.py (hunk 1 above)
Comment (cehbrecht) [reply birdy/client/base.py]: Yes ... copy+paste form the docs:
https://docs.python.org/3/library/collections.html#collections.somenamedtuple._make

So, I thought to take the example code ...

File: birdy/utils.py (hunk 2 above)
Comment (cehbrecht) [reply birdy/utils.py]: ok, I will update.

Comment (cehbrecht) [reply birdy/utils.py]: done.

File: birdy/client/base.py (hunk 1 above)
Comment (cehbrecht) [reply birdy/client/base.py]: accept?

Comment (huard) [reply birdy/client/base.py]: I'd prefer using the public method.

**Pull request merged**